EU4toV2/Source/EU4World/Country/EU4CustomColors.h

```diff
@@ -19,28 +19,44 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE. */
 
 
-
-#ifndef CUSTOM_FLAG_MAPPER
-#define CUSTOM_FLAG_MAPPER
-
+#ifndef EU4_CUSTOMCOLORS_H_
+#define EU4_CUSTOMCOLORS_H_
 
 
+#include "Color.h"
+#include "Date.h"
```

Comment:
I don't see date used anywhere, so can probably be removed.

Reply:
done